cognite/client/_api/data_modeling/instances.py

```diff
@@ -1655,13 +1696,29 @@ def list(
             List instances with a view as source:
 
                 >>> from cognite.client.data_classes.data_modeling import ViewId
-                >>> instance_list = client.data_modeling.instances.list(sources=ViewId("mySpace", "myView", "v1"))
+                >>> my_view = ViewId("mySpace", "myView", "v1")
+                >>> instance_list = client.data_modeling.instances.list(sources=my_view)
 
-            Convert instances to pandas DataFrame with expanded properties:
+            Convert instances to pandas DataFrame with expanded properties (``expand_properties=True``).
+            This will add the properties directly as dataframe columns. Specifying ``camel_case=True``
+            will convert the basic columns to camel case (e.g. externalId), but leave the property names as-is.
 
-                >>> df = instance_list.to_pandas(expand_properties=True, camel_case=True)
-                >>> # expand_properties=True will add the properties directly as dataframe columns
-                >>> # camel_case=True will convert the DataFrame column names to camel case (e.g. externalId)
+                >>> df = instance_list.to_pandas(
+                ...     expand_properties=True,
+                ...     camel_case=True,
+                ... )
+
+            To get debug information in the response, pass ``DebugParameters``:
+
+                >>> from cognite.client.data_classes.data_modeling.debug import DebugParameters
+                >>> res = client.data_modeling.instances.list(
+                ...     debug=DebugParameters(
+                ...         emit_results=False,
+                ...         timeout=20*1000,  # 20 sec timeout
+                ...         profile=True
+                ...     ),
+                ...     sources=my_view,
+                ... )
```

Comment:
Suggest showing how to access it.
```suggestion
                ... )
                >>> print(res.debug)
```